src/GoogleAnalytics/Client.php

```diff
@@ -57,14 +57,22 @@ public function getSegments(): array
 
     public function getAccountProperties(): array
     {
-        $response = $this->api->request(sprintf('%s?pageSize=200', self::ACCOUNT_PROPERTIES_URL));
-        $body = json_decode($response->getBody()->getContents(), true);
+        $accountSummaries = [];
 
-        if (isset($body['accountSummaries'])) {
-            return array_filter($body['accountSummaries'], fn(array $v) => isset($v['propertySummaries']));
-        }
+        do {
+            $url = sprintf('%s?pageSize=200', self::ACCOUNT_PROPERTIES_URL);
```

Comment:
prosím nějjaký test na to... zkus ten pageSize dát spíš jako parametr s defaultní hodnotou přes konstantu a v testu tu přepsat na 1 a otestovat to